packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -60,12 +65,26 @@ export class GutenbergEditorPage extends BaseContainer {
 	}
 
 	/**
-	 * Enters text into the title block.
+	 * Enters the text into the title block and verifies the result.
 	 *
 	 * @param {string} title Text to be used as the title.
 	 * @returns {Promise<void>} No return value.
+	 * @throws {assert.AssertionError} If text entered and text read back do not match.
 	 */
 	async enterTitle( title: string ): Promise< void > {
+		const sanitizedTitle = title.trim();
+		await this.setTitle( sanitizedTitle );
+		const readBack = await this.getTitle();
+		assert( readBack === sanitizedTitle );
```

Comment:
`assert.strictEqual(readBack, sanitizedTitle)` will give you a better error message in case it fails